table/get_context.cc

```diff
@@ -88,10 +89,33 @@ void GetContext::SaveValue(const Slice& value, SequenceNumber seq) {
 }
 
 void GetContext::RecordCounters(Tickers ticker, size_t val) {
-  if (ticker == Tickers::TICKER_ENUM_MAX) {
-    return;
+  for (auto it = ticker_pairs_.begin(); it != ticker_pairs_.end(); ++it) {
```

Comment:
I don't have intuition why this PR is faster overall -- before we had an array lookup here, and now it's a for-loop. Have you measured it? Make sure to use a fairly large DB as then a single Get() will have to execute this for-loop many times.